testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+
+    @Test
+    public void pass_if_float_values_are_not_equal() {
+        final float expected = 0.0f;
+        final float actual = 1.0f;
+        final float lessThanValuesDifference = Math.abs(expected - actual) - 0.1f;
+        Verify.assertNotEquals(expected, actual, lessThanValuesDifference);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_bool_values_are_equal() {
+        Verify.assertNotEquals(true, true);
+    }
+
+    @Test
+    public void pass_if_bool_values_are_not_equal() {
+        Verify.assertNotEquals(true, false);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_byte_values_are_equal() {
+        Verify.assertNotEquals((byte) 0, (byte) 0);
+    }
+
+    @Test
+    public void pass_if_byte_values_are_not_equal() {
+        Verify.assertNotEquals((byte) 0, (byte) 1);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_char_values_are_equal() {
+        Verify.assertNotEquals('a', 'a');
+    }
+
+    @Test
+    public void pass_if_char_values_are_not_equal() {
+        Verify.assertNotEquals('a', 'b');
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_short_values_are_equal() {
+        Verify.assertNotEquals((short) 0, (short) 0);
+    }
+
+    @Test
+    public void pass_if_short_values_are_not_equal() {
+        Verify.assertNotEquals((short) 0, (short) 1);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_object_is_not_instance_of_specified_type() {
+        Verify.assertInstanceOf(Integer.class, "");
+    }
+
+    @Test
+    public void pass_if_object_is_instance_of_specified_type() {
+        final String instance = "";
+        Verify.assertInstanceOf(instance.getClass(), instance);
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_object_is_instance_of_specified_type() {
+        final String instance = "";
+        Verify.assertNotInstanceOf(instance.getClass(), instance);
+    }
+
+    @Test
+    public void pass_if_object_is_not_instance_of_specified_type() {
+        Verify.assertNotInstanceOf(Integer.class, "");
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_not_empty() {
+        Verify.assertIterableEmpty(FluentIterable.of(1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_null_in_iterable_empty() {
+        Verify.assertIterableEmpty(null);
+    }
+
+    @Test
+    public void pass_if_iterable_is_empty() {
+        Verify.assertIterableEmpty(FluentIterable.of());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_not_empty() {
+        Verify.assertEmpty(Collections.singletonMap(1, 1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_assert_empty() {
+        Verify.assertEmpty((Map) null);
```

Comment:
@armiol PTAL that there are overloaded functions, so cast is required to prevent ambiguous method call. In places where cast is not required, i don’t use it.